gnomad_qc/v3/resources/sample_qc.py

```diff
@@ -73,55 +116,124 @@ def get_v3_relatedness_annotated_ht() -> Table:
+relatedness = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_qc_mt_v2_sites_relatedness.ht") for release in RELEASES}
+)
 
-# Sex imputation results
-v3_sex = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_sex.ht')
+# Sex imputation results	
+sex = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_sex.ht") for release in RELEASES}
+)
 
 # Samples to drop for PCA due to them being related
-pca_related_samples_to_drop = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_related_samples_to_drop_for_pca.ht')
+pca_related_samples_to_drop = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_related_samples_to_drop_for_pca.ht") for release in RELEASES}
+)
+
+# Related samples to drop for release	
+release_related_samples_to_drop = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_related_release_samples_to_drop.ht") for release in RELEASES}
+)
 
-# Related samples to drop for release
-release_related_samples_to_drop = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_related_release_samples_to_drop.ht')
+# Inferred sample populations	
+pop = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_pop.ht") for release in RELEASES}
+)
 
-# Inferred populations
-pop = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_pop.ht')
-pop_tsv_path = 'gs://gnomad/sample_qc/temp/genomes_v3/gnomad_v3_RF_pop_assignments.txt.gz'
-pop_rf_path = 'gs://gnomad/sample_qc/temp/genomes_v3/gnomad_v3_pop.RF_fit.pickle'
+
+def pop_tsv_path(version: str = CURRENT_RELEASE) -> str:
+    """
+    Path to tab delimited file indicating inferred sample populations
+
+    :param version: Version of sample QC path to return
+    :return: String path to sample populations
+    """
+    return f"gs://gnomad/sample_qc/temp/genomes_v{version}/gnomad_v{version}_RF_pop_assignments.txt.gz"
+
+
+def pop_rf_path(version: str = CURRENT_RELEASE) -> str:
+    """
+    Path to RF model used for inferring sample populations
+
+    :param version: Version of sample QC path to return
+    :return: String path to sample pop RF model
+    """
+    return f"gs://gnomad/sample_qc/temp/genomes_v{version}/gnomad_v{version}_pop.RF_fit.pickle"
 
 # Hard-filtered samples
-hard_filtered_samples = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_hard_filtered_samples.ht')
+hard_filtered_samples = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_hard_filtered_samples.ht") for release in RELEASES}
+)
 
-# Results of running population-based metrics filtering
+# Results of running population-based metrics filtering	    :param version: Version of sample QC path to return
```

Comment:
Looks like `:param version: ...` was unintentionally pasted here? Same thing for `release_samples_rankings` below.

Reply:
Yes, thank you -- this slipped by me.